modules/nw-sriov-hwol-about-hardware-offloading.adoc

```diff
@@ -26,4 +26,6 @@ Only the following two communication types are supported:
 In all cases, hardware offloading takes place only when those pods and services are assigned to nodes that have a compatible SmartNIC.
 Suppose, for example, that a pod on a node with hardware offloading tries to communicate with a service on a regular node.
 On the regular node, all the processing takes place in the kernel, so the overall performance of the pod-to-service communication is limited to the maximum performance of that regular node.
-Hardware offloading is not compatible with DPDK applications.
\ No newline at end of file
+Hardware offloading is not compatible with DPDK applications.
+
+Enabling hardware offloading on a node, but not configuring pods to use it can result in decreased throughput performance for pod traffic. You cannot configure hardware offloading for pods that are managed by {product-title}.
```

Comment:
```suggestion
Enabling hardware offloading on a node, but not configuring pods to use, it can result in decreased throughput performance for pod traffic. You cannot configure hardware offloading for pods that are managed by {product-title}.
```